settings.yml

```diff
@@ -1,7 +1,39 @@
 custom_new_topic_text:
-  default: ""
-  json_schema: '{"type":"array","uniqueItems":true,"items":{"type":"object","properties":{"filter":{"type":"string","minLength":1,"description":"Category or tag name (note: tags take precedence)"},"icon":{"type":"string","default":"plus","description":"Font awesome icon name for the New Topic button"},"button_text":{"type":"string","description":"New Topic button"},"composer_action_text":{"type":"string","description":"Composer action name (Create a new topic by default)"},"composer_button_text":{"type":"string","description":"Composer post button"}},"additionalProperties":false}}'
-
+  default: >-
+    []
```

Comment:
```suggestion
  default: []
```

Reply:
Thanks for the review @lis2!  It would cause an error if I did that, but I can do:

```
default: "[]"
```